plugins/osdn/common.go

```diff
@@ -29,6 +29,7 @@ type OvsController struct {
 	Registry        *Registry
 	localIP         string
 	localSubnet     *api.Subnet
+	clusterNet      *net.IPNet
```

Comment:
> Specifically I'm wondering about the order of execution between StartMaster and watchNodes.

watchNodes() only gets called from within SubnetStartMaster(), which gets called from within StartMaster() after oc.clusterNet is set, so this should be ok